hack/verify-flags-underscore.py

```diff
@@ -62,8 +62,6 @@ def get_all_files(rootdir):
             dirs.remove('third_party')
         if '.git' in dirs:
             dirs.remove('.git')
-        if '.make' in dirs:
-            dirs.remove('.make')
         if 'BUILD' in files:
```

Comment:
TODO(bentheelder): we can probably drop this BUILD logic now ...

Reply:
I found a few refs to bzl files, and so forth - probably can lean those up too

Reply:
Filed https://github.com/kubernetes/kubernetes/pull/112864
Mostly found BUILD, only one .bzl 👀